tests/test_ptc.py

```diff
@@ -193,6 +205,49 @@ def test_covAstier(self):
             for v1, v2 in zip(varStandard[amp], ptc.finalVars[amp]):
                 self.assertAlmostEqual(v1/v2, 1.0, places=1)
 
+            # Check that the PTC turnoff is correctly computed.
+            # This will be different for the C:0,0 amp.
+            if amp == 'C:0,0':
+                self.assertAlmostEqual(ptc.ptcTurnoff[amp], ptc.rawMeans[ampName][-3])
+            else:
+                self.assertAlmostEqual(ptc.ptcTurnoff[amp], ptc.rawMeans[ampName][-1])
+
+            # Test that all the quantities are correctly ordered and have
+            # not accidentally been masked.
+            for i, extractPtc in enumerate(resultsExtract.outputCovariances[::2]):
```

Comment:
Another indexing question to make sure I'm not confused:  This is skipping the "dummy" entries, correct?